staking-miner-playground/runtime/src/lib.rs

```diff
@@ -343,7 +332,7 @@ impl<const PERIOD: BlockNumber> ShouldEndSession<BlockNumber>
 	}
 }
 
-const SESSION: BlockNumber = 1 * MINUTES;
+const SESSION: BlockNumber = 3 * MINUTES;
```

Comment:
Because the trimming is slower with "pre-trimming", I increased the SignPhase a bit...